internal/services/mssql/mssql_virtual_machine_resource_test.go

```diff
@@ -631,10 +631,13 @@ resource "azurerm_virtual_machine_data_disk_attachment" "test" {
 resource "azurerm_mssql_virtual_machine" "test" {
   virtual_machine_id = azurerm_virtual_machine.test.id
   sql_license_type   = "PAYG"
-
+  
+  assessment_settings {}
```

Comment:
As this is a new block of property, maybe it worth writing a separate test for it with testing the update scenario as well.